packages/framework/src/ShopsysFrameworkBundle.php

```diff
@@ -29,5 +31,9 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new RegisterPluginDataFixturesCompilerPass());
         $container->addCompilerPass(new RegisterProductFeedConfigsCompilerPass());
         $container->addCompilerPass(new LazyRedisCompilerPass());
+        $environment = $container->getParameter('kernel.environment');
+        if ($environment !== EnvironmentType::PRODUCTION) {
```

Comment:
You are right, at the time I was thinking it might be necessary because of the possible tests but now I think I won't need it in unit tests at all.

Fixed in ea2f13eb5